onnxruntime/core/platform/env.h

```diff
@@ -120,7 +140,7 @@ class Env {
   virtual int GetNumCpuCores() const = 0;
 
   // This function doesn't support systems with more than 64 logical processors
-  virtual std::vector<size_t> GetThreadAffinityMasks() const = 0;
+  virtual std::vector<ThreadOptions::ThreadAffinity> GetThreadAffinityMasks() const = 0;
```

Comment:
>hreadOptions::ThreadAffinity> [](http://example.com/codeflow?start=23&length=29)

Shall we define this by a global "using" out of "ThreadOptions", the type looks to be referred by multiple classes/structs.